.github/workflows/testing-mac.yml

```diff
@@ -34,19 +34,66 @@ jobs:
-      #   uses: actions/cache@v2
-      #   id: cache-datasets
-      #   with:
-      #     path: /home/runner/work/nwb-conversion-tools/nwb-conversion-tools/ephy_testing_data
-      #     key: ecephys-datasets-3-${{ steps.ephys.outputs.HASH_EPHY_DATASET }}
-      #     restore-keys: ecephys-datasets-3-${{ steps.ephys.outputs.HASH_EPHY_DATASET }}
-      # - name: Run pytest with GIN data
-      #   if: steps.cache-datasets.outputs.cache-hit == true
-      #   run: pytest
-      - name: Run pytest without GIN data
-        # if: steps.cache-datasets.outputs.cache-hit == false
-        run: pytest tests/test_internals
+      - name: Install git-annex
+        run: brew install git-annex
+      - name: Install datalad
+        run: |
+          pip install datalad
+          git config --global user.email "CI@example.com"
+          git config --global user.name "CI Almighty"
+
+      - name: Get behavior_testing_data current head hash
+        id: behavior
+        run: echo "::set-output name=HASH_BEHAVIOR_DATASET::$(git ls-remote https://gin.g-node.org/CatalystNeuro/behavior_testing_data.git HEAD | cut -f1)"
+      - name: Cache behavior dataset - ${{ steps.behavior.outputs.HASH_BEHAVIOR_DATASET }}
+        uses: actions/cache@v2
+        id: cache-behavior-datasets
+        with:
+          path: ./behavior_testing_data
+          key: behavior-datasets-00-${{ steps.behavior.outputs.HASH_behavior_DATASET }}
+      - name: Force GIN behavior download
+        if: steps.cache-behavior-datasets.outputs.cache-hit == false
+        run: datalad install -rg https://gin.g-node.org/CatalystNeuro/behavior_testing_data
+
+      - name: Get ephy_testing_data current head hash
+        id: ecephys
+        run: echo "::set-output name=HASH_EPHY_DATASET::$(git ls-remote https://gin.g-node.org/NeuralEnsemble/ephy_testing_data.git HEAD | cut -f1)"
+      - name: Cache ephys dataset - ${{ steps.ecephys.outputs.HASH_EPHYS_DATASET }}
+        uses: actions/cache@v2
+        id: cache-ecephys-datasets
+        with:
+          path: ./ephy_testing_data
+          key: ecephys-datasets-9-${{ steps.ecephys.outputs.HASH_EPHY_DATASET }}
+          restore-keys: ecephys-datasets-00-${{ steps.ecephys.outputs.HASH_EPHY_DATASET }}
+      - name: Force GIN ecephys download
+        if: steps.cache-ecephys-datasets.outputs.cache-hit == false
+        run: |
+          datalad install https://gin.g-node.org/NeuralEnsemble/ephy_testing_data
+          cd ephy_testing_data
+          datalad get -r ./neuralynx/Cheetah_v5.7.4/original_data/
+          datalad get -r ./neuroscope/test1/
+          datalad get -r ./openephysbinary/v0.4.4.1_with_video_tracking/
+          datalad get -r ./blackrock/
+          datalad get -r ./intan/
+          datalad get -r ./spikegadgets/
+          datalad get -r ./spikeglx/Noise4Sam_g0/Noise4Sam_g0_imec0/
+          datalad get -r ./phy/phy_example_0/
+          datalad get -r ./axona/
+          datalad get -r ./cellexplorer/
```

Comment:
Is there a way we can incorporate these calls into the .py files so we don't have to keep updating this every time we add a new format?

Reply:
Incorporating them into the test files simply passes the buck by requiring the extra line to be added somewhere else; an explicit pointer from a format to a GIN location would always have to be specified at some level.

It's on @h-mayorquin low-priority to-do list however to centralize these sets of lines into a single place so it only has to be updated one per format addition

Although, what we can probably start doing, now that we support such a large number of neo formats, is just call a single line of  `datalad -rg name_of_dataset` and that will handle both issues.